trk/src/Screens/SetDetail/index.js

```diff
@@ -14,6 +15,9 @@ function SetDetail(props) {
   const [tapCount, setTapCount] = useState(0);
   const [climbImageUrl, setClimbImageUrl] = useState(null);
   const [setterImageUrl, setSetterImageUrl] = useState(null);
+  const [feedbackList, setFeedbackList] = useState([]);
+
+  const { getCommentsBySomeField } = CommentsApi();
 
```

Comment:
1. initialize empty feedbackList
2. import the function to get comments by desired field (in this case it will be by climb id)